spec/rails_helper.rb

```diff
@@ -0,0 +1,77 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
+require_relative '../config/environment'
+# Prevent database truncation if the environment is production
+abort('The Rails environment is running in production mode!') if Rails.env.production?
+require 'rspec/rails'
+# Add additional requires below this line. Rails is not loaded until this point!
+
+# Requires supporting ruby files with custom matchers and macros, etc, in
+# spec/support/ and its subdirectories. Files matching `spec/**/*_spec.rb` are
```

Comment:
- [ ] While you did well adding the rails_helper file, kindly note that your tests will still not run. Note that you are requiring a "spec_helper" file which does not exist which will cause your tests to not run.

**This is the error I am facing:**

![image](https://github.com/geekelo/cash-in-out/assets/86318284/4fad9553-e858-4e41-94df-c72e53dda0a1)

To fix the error above you need to have a spec_helper.rb file as well.

Adding the above mentioned file will enable your tests to run just as in the screenshot linked below:

![image](https://github.com/geekelo/cash-in-out/assets/86318284/5c54d0ee-35bf-423b-9d6c-c516d9fae93f)